common-auth/pom.xml

```diff
@@ -103,6 +103,12 @@
         </resource>
       </resources>
         <plugins>
+            <plugin>
```

Comment:
@vion1 Can you put a comment with details on why this was added so that others would know the reason. As you mentioned in the bug, this is because of apacheds-jdbm1-2.0.0-M2.bundle not available now.

Reply:
apacheds-jdbm1-2.0.0-M2.bundle is no longer available on nexus so the build will fail.   This fix will download the correct artifact apacheds-jdbm1-2.0.0-M2.jar from nexus.

Reply:
@vion1 Can you put the above comment in the file at that line(which is 106) so that whoever is looking at the file can know the reason?